node_modules/oae-core/filepreview/js/filepreview.js

```diff
@@ -80,6 +80,20 @@ define(['require', 'jquery', 'oae.core'], function(require, $, oae) {
         };
 
 
+        /////////////////////
+        //  VIDEO PREVIEW  //
+        /////////////////////
+
+        /**
+         * Render a preview of a video file
+         */
+        var renderVideoPreview = function() {
```

Comment:
I think this will be something that's happening on the Hilary side, will look into it.